src/pages/tutorials/javascript-libraries.md

```diff
@@ -0,0 +1,28 @@
+---
+title: JavaScript Libraries
+description: How to use Esri Leaflet with modern JavaScript libraries and frameworks.
+layout: tutorials.hbs
+---
+
+# {{ page.data.title }}
+
+{{ page.data.description }}
+
+#### Introduction
+
+Leaflet and Esri Leaflet can be used with all the modern JavaScript libraries like React, Svelte, Vue, and Angular. In general, you'll use the functionality in your framework to get a reference to a DOM Node. For example, `refs` in React or the `use:` directive in Svelte. Then you can import Leaflet and Esri Leaflet using modern ES6 style imports:
```

Comment:
I'm on the fence about providing (and then having to maintain) outside framework docs links, but what do you think about adding these in? Totally fine if you want to disregard.

```suggestion
Leaflet and Esri Leaflet can be used with all the modern JavaScript libraries like React, Svelte, Vue, and Angular. In general, you'll use the functionality in your framework to get a reference to a DOM Node. For example, [`refs` in React](https://reactjs.org/docs/refs-and-the-dom.html) or [the `use:` directive in Svelte](https://svelte.dev/docs#use_action). Then you can import Leaflet and Esri Leaflet using modern ES6 style imports:
```